src/app.js

```diff
@@ -118,6 +124,23 @@ function getOptions() {
   return options;
 }
 
+/**
```

Comment:
@benfrancis picking up from #268. `rulesEngineConfigure` exists because `Router.configure` runs before we know whether the server will be `http` or `https`. If we're running the tests then we also don't know the server's port. This is a problem because `node-fetch` only works with a complete URL.

It's unnecessary for this function to also issue the JWT. The `rules-engine/index.js` file can use JSONWebToken directly pretty easily. I'm going to switch it over to that approach so it's more clear why this code exists.